### ISSUE TITLE
LCF-glove精度的疑问

### ISSUE THREAD

**Issue by caucsunjiahui** [yangheng95/PyABSA#7]
尊敬的作者您好！
      在Laptop数据集上，按照您给出的配置参数，config1进行实验，seed选择从0-5，准确率分别为72.57、74.45、73.98、73.82、72.72、73.04，均未能达到76.02的精度，想请教一下我是否在哪个步骤疏忽了，谢谢！
"config1": {
    "model_name": "lcf_glove",
    "dataset": "laptop",
    "use_single_bert": null,
    "optimizer": "adam",
    "initializer": "xavier_uniform_",
    "learning_rate": 0.001,
    "dropout": 0,
    "l2reg": 0.00001,
    "num_epoch": 10,
    "batch_size": 16,
    "log_step": 5,
    "logdir": "log",
    "bert_dim": 768,
    "pretrained_bert_name": "",
    "max_seq_len": 80,
    "polarities_dim": 3,
    "hops": 3,
    "SRD": 6,
    "local_context_focus": "cdm"
  },

**Comment by yangheng95** [comment]
LCF-GloVe模型的代码几经修改，原始代码已经无法找到了，非常抱歉。建议您关注LCF-BERT或者我们的其他模型
